de_DE/home/src/main/resources/rdf/tbox/everytime/initialTBoxAnnotations_de_DE.nt

```diff
@@ -1,865 +0,0 @@
-<http://purl.org/ontology/bibo/section> <http://www.w3.org/2000/01/rdf-schema#label> "Abschnitt"@de-DE .
```

Comment:
Why are you deleting this file from the `de_DE` set, while it remains in the other languages... except `fr_CA`?

Reply:
In https://github.com/vivo-project/VIVO-languages/commit/b76f5572362f7445c47bc97adf75e29a9490a873 I moved the now missing german annotations file to the tbox filegraph directory and converted it into turtle and renamed it to vivo_de_DE.ttl. I did so because the structure within the french language tree was my orientation. In this commit I just moved the spanish and portuguese language file to the filegraph folder without converting nor renaming. Despite the fact that this is enough to get a working VIVO installation I just wonder now if - for sake of a unified structure within the different language folders - a) we should stuck to a naming convention and b) should convert all files to ttl.